Resources/Localizations/en.lproj/Localizable.strings

```diff
@@ -801,16 +801,19 @@
 "navigate_point_format_DMS" = "DDD MM SS.SS";
 "navigate_point_format_OLC" = "Open Location Code (OLC)";
 "navigate_point_format_UTM" = "UTM";
+"navigate_point_format_MGRS" = "MGRS";
 "navigate_point_format_unknown" = "Unknown format";
 "navigate_point_northing" = "Northing";
 "navigate_point_easting" = "Easting";
 "navigate_point_zone" = "Zone";
 "navigate_point_olc" = "Open Location Code";
 "navigate_point_olc_short" = "OLC";
+"navigate_point_mgrs" = "MGRS";
```

Comment:
Please remove this line and use the one above, we try not to duplicate string resources.

Reply:
Done with 31fe5a0